pkg/detector/detector.go

```diff
@@ -1272,15 +1267,15 @@ func (d *ResourceDetector) HandleClusterPropagationPolicyDeletion(policyName str
 func (d *ResourceDetector) HandlePropagationPolicyCreationOrUpdate(policy *policyv1alpha1.PropagationPolicy) error {
 	// If the Policy's ResourceSelectors change, causing certain resources to no longer match the Policy, the label marked
 	// on relevant resource template will be removed (which gives the resource template a change to match another policy).
-	err := d.cleanPPUnmatchedResourceBindings(policy.Namespace, policy.Name, policy.Spec.ResourceSelectors)
+	err := d.cleanupUnmatchedRBsWithPP(policy.Namespace, policy.Name, policy.Spec.ResourceSelectors)
 	if err != nil {
 		return err
 	}
 
 	// When updating fields other than ResourceSelector, should first find the corresponding ResourceBinding
 	// and add the bound object to the processor's queue for reconciliation to make sure that
 	// PropagationPolicy's updates can be synchronized to ResourceBinding.
-	resourceBindings, err := d.listPPDerivedRB(policy.Namespace, policy.Name)
```

Comment:
I more tend to the original name.

Reply:
The relationship between RB and PP can't be described as a derived relationship, right?

Reply:
hard to say.